JASP-Desktop/widgets/boundqmltextarea.cpp

```diff
@@ -19,6 +19,7 @@
 #include "boundqmltextarea.h"
 #include "../analysis/analysisform.h"
 #include "qmllistviewtermsavailable.h"
+#include "../../JASP-Common/r_functionwhitelist.h"
```

Comment:
I think this can just be `#include "r_functionwhitelist.h"` because JASP-Common is already an `INCLUDEPATH`.